mbuild/formats/hoomd_forcefield.py

```diff
@@ -70,9 +70,34 @@ def create_hoomd_forcefield(
     ----
     If you pass a non-parametrized pmd.Structure, you will not have
     angle, dihedral, or force field information. You may be better off
-    creating a hoomd.Snapshot
-    Reference units should be expected to convert parmed Structure units :
-        angstroms, kcal/mol, and daltons
+    creating a hoomd.Snapshot.
+
+    Note about units
+    ----------------
+    This method operates on a Parmed.Structure object where the units used
+    differ from those used in mBuild and Foyer, which may have been used
+    when creating the typed Parmed.Structure.
+
+    The default units used when writing out the HOOMD Snapshot are:
+    Distance (Angstrom)
+    Mass (Dalton)
+    Energy (kcal/mol)
+
+    If you wish to convert this unit system to another, you can use the
+    reference parameters (ref_distance, ref_mass, ref_energy).
+    The values used here should be expected to convert from the Parmed
+    Structure units (above) to your desired units.
+    The Parmed.Structure values are divided by the reference values.
+
+    Examples:
+        To convert the enregy units from kcal/mol to kj/mol:
+            use ref_energy = 0.2390057 (kcal/kj)
+        To convert the distance units from Angstrom to nm:
+            use ref_distance = 10 (angstroms/nm)
+
+    You can also use the auto_scale parameter to convert distance, energy
```

Comment:
You might want to give an example of this as well. I'd move line 98 up above the Examples section, then make a full new:
```python
Examples
-----------
```
Section with all of these individual examples. See something like [this](https://github.com/mosdef-hub/mbuild/blob/d051e79f24d73917be72f7cd7a09081a4b748170/mbuild/compound.py#L509) as a format.

Reply:
I consolidated the examples section here.  I see what you're saying about the examples in that other file. Do you think I should write out a full example from imports to creating an mb.Compound, etc.. or just leave these examples specific to picking a value for the reference parameters.